borg/key.py

```diff
@@ -105,7 +105,8 @@ def encrypt(self, data):
 
     def decrypt(self, id, data):
         if data[0] != self.TYPE:
-            raise IntegrityError('Chunk %s: Invalid encryption envelope' % bin_to_hex(id))
+            id_str = bin_to_hex(id) if id is not None else 'None'
```

Comment:
Something like `(unknown)` instead of `None`, due to: `Chunk None: Invalid....`?